package.json

```diff
@@ -87,7 +87,7 @@
     "@qwant/merge-po": "file:local_modules/merge-po",
     "@qwant/nconf-builder": "file:local_modules/nconf_builder",
     "@qwant/po-js": "file:local_modules/po-js",
-    "@qwant/qwant-basic-gl-style": "git+https://github.com/QwantResearch/qwant-basic-gl-style.git#45e60be",
+    "@qwant/qwant-basic-gl-style": "git+https://github.com/QwantResearch/qwant-basic-gl-style.git#94f3fbc",
```

Comment:
Could you update the commit to use the latest master, now that https://github.com/QwantResearch/qwant-basic-gl-style/pull/82 is merged?

And `package-lock.json` needs to be committed too.